### DISCUSSION TITLE
IoT-LTSC-Like - File Explorer Quick Access

### DISCUSSION THREAD

**Discussion by HC1Gunner** [memstechtips/UnattendedWinstall#105] [Q&A]
Does the IoT-LTSC-Like autounattened file mess with file explorer's "quick access" on its left panel. Since my clean install with this xml file, my Quick Access has not populated one single folder.
I went to options, on the general tab did a Restore Defaults, and on the view tab, Reset Folders, this fixed nothing.
I then created a new User account, logged on, did a bunch of downloads using only two folders, and opened a couple of the same folders a bunch of times, and it never populates the Quick Access area.

**Comment by memstechtips** [comment]
Hey there and sorry for the late reply.

I'm not aware of anything that specifically removed the quick access from the Navigation Pane.

But, since the creation of this issue I have released a new version of the XML file and a major update to [Winhance](https://winhance.net/), I recommend that you watch [this video](https://youtu.be/lrq3ph3xi50) which provides a full demo of the current version along with how to use the new version of WIMUtil and how to install the drivers etc. You now also have the ability to create an autounattend.xml file based on your selections in Winhance, so you have more control over which apps are removed and which settings are applied.

I have also created [this video](https://youtu.be/sCd1TI6AbOI) about v3.0.0 of UnattendedWinstall, I suggest that you watch it too if you're interested.

I'm closing this issue now, but if you need more help with the same issue, feel free to reopen it.
